src/main/java/com/gildedrose/GildedRose.java

```diff
@@ -1,6 +1,23 @@
 package com.gildedrose;
 
 class GildedRose {
+    public enum ItemName {
+
+        AGED_BRIE("Aged Brie"), BACKSTAGE("Backstage passes to a TAFKAL80ETC concert"),
```

Comment:
`BACKSTAGE` is not a good short form since it does not capture what the item actually is. Better alternatives might be `PASSES` or `BACKSTAGE_PASSES`.